ClientRuntimes/NodeJS/lib/validate.js

```diff
@@ -52,19 +52,18 @@ exports.isValidUuid = function(uuid, callback) {
 };
 
 /**
-* Creates a anonymous function that check if a given key is base 64 encoded.
+* Validates if the given string is base64 encoded
 *
 * @param {string} key The key to validate.
-* @return {function}
+* @return {bool} 
 */
 exports.isBase64Encoded = function (key) {
   var isValidBase64String = key.match('^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$');
-
+  var result = false;
   if (isValidBase64String) {
```

Comment:
please simplify.
The entire function only need one line:
   return str,match(.....);